guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
 
 # Open design
 
-**Open design is an alternative design and development method based on the free exchange of information.**
+**Open design welcomes the free exchange of ideas, knowledge, processes, and tools. It empowers by lifting barriers to innovation. It opens the door to scaling design projects by distributing adaptable resources.**
 
-By balancing the independence of individual designers and the collective power of collaboration, open design enables and promotes the convictions and passions of designers working towards the benefit of society.
+## A balance of independence and collaboration
 
-Open design can manifest in many different ways. You might:
+Open source projects are often driven by global communities. These hubs of learning and building allow you to carve an individual niche. Align yourself to projects based on your passions, talents, interests, and beliefs. Your work becomes a voluntary public contribution toward common goals. Some projects are also supported through grants and donations.
 
-* help design the interface of your favorite open-source project
-* collaborate with people and projects in areas related to specific topics of interest
-* create a design resource library like an icon set or a UI kit
-* help test an open-source product and provide feedback
-* [contribute](/guide/contribute/) to the Bitcoin Design Guide
+Open design communities can have multidisciplinary teams with designers, developers, copywriters, translators, project managers.... These diverse groups can develop stronger solutions through varied points of view.
 
-Whatever you choose to do, open design projects are voluntary public contributions. From this work, a network of people and projects emerge connected by personal relationships, specialized communities, and sharing of work. This is different from corporate structures and requires individuals to think and act more independently.
+## Helpful skills
 
-In open-source software, public review and debate lead to better products and services. Open-source development has existed for decades, while open design is not widely known or practiced. Designing for bitcoin products, especially for newcomers in an open design setting, can be disorienting and require adjustments to how they work and collaborate. No longer is work the exclusive product of them or a small, localized team; instead, it becomes part of a public conversation.
+- Ethics
+- Personal responsibility
+- Independence & self-will
```

Comment:
I believe "self-will" is when someone is stubborn (meaning they do what they want no matter what anyone says). Do you mean "self-motivation" in that someone is driven on their own?

Reply:
Agreed. I'm actually not familiar with the term "self-will" but "self-motivation" sounds right.

Reply:
Changed to self-motivation in https://github.com/BitcoinDesign/Guide/commit/453525b8045f957d98b6af6558586f2b80b7e200